pylib/mozhginfo/mozhginfo/push.py

```diff
@@ -0,0 +1,62 @@
+class Push(object):
```

Comment:
You will need to add __repr__ and __cpm__ to the class for sorting and printing.